backend/back/src/database.ts

```diff
@@ -92,6 +93,7 @@ export const DATA_SOURCE_OPTIONS: DataSourceOptions = {
 		CreateIssueGithubReaction1698062010534,
 		CreateGoogleSlideInPresentationArea1698150990404,
 		UpdateServiceImagesToBrandFetchSVGs1698161840408,
+		OnIssueCreatedGithubAction1698179108023,
```

Comment:
Please follow the name convention for the migrations "Create<YourAreaName>Area"

Reply:
It doesn't really make sense for actions

Reply:
why ?

Reply:
You want me to call it CreateOnIssueGithubAction ???

Reply:
CreateGithubOnIssueCreationArea

Reply:
Done